client/src/main/java/io/pravega/client/stream/impl/ModelHelper.java

```diff
@@ -172,6 +174,34 @@ public static final PravegaNodeUri encode(final NodeUri uri) {
         return result;
     }
 
+    /**
+     * Returns the status of Ping Transaction.
+     *
+     * @param status     PingTxnStatus object instance.
+     * @param logString Description text to be logged when ping transaction status is invalid.
+     * @return Transaction.PingStatus
+     */
+    @SneakyThrows
```

Comment:
PingFailedException doesn't need to be sneaky throws. It can just be a declared exception.

Reply:
ok, removed sneaky throws. Catching the exception and wrapping it with Completion exception.